bundle/edu.gemini.horizons.api/src/main/scala/edu/gemini/horizons/server/backend/HS2.scala

```diff
@@ -280,12 +280,18 @@ object HorizonsService2 {
               List(Row(HorizonsDesignation.AsteroidNewStyle(m.group(1)) : HorizonsDesignation.Asteroid, m.group(1)))
             } \/> "Could not match '(2016 GB222)' header pattern."
 
+          // Single result with form: JPL/HORIZONS        418993 (2009 MS9)            2016-Sep-07 18:23:54
+          lazy val case4 =
+            """  +\d+\s+\((.+?)\)  """.r.findFirstMatchIn(header).map { m =>
```

Comment:
@cquiroz It's explicitly looking for two spaces because there is leading and trailing text in the header line with space padding (the text we're looking for is centered). So it could really be two or more but `  ` is easier to read/write than `\s{2,}`.